src/shared/components/download/downloadUtils.ts

```diff
@@ -41,6 +41,7 @@ const reSubsequenceFrom = new RegExp(
   `(${reUniProtKBAccession.source})${reSubsequence.source}`,
   'i'
 );
+const reProteomeId = /(upid|proteome):UP\d+/;
```

Comment:
Noticed this regex twice in the PR - would be handy to have it in one place should it ever need updating.